docs/source/guides/demo/liquor/__init__.py

```diff
@@ -0,0 +1,14 @@
+from demo import PWD
+from pandas import read_csv
+from os.path import join
+
+PWD = join(PWD, 'liquor')
+
+def load_sample():
+    df = read_csv(join(PWD, "Iowa_Liquor_Sales.csv"))
+    # df = df.drop(columns=df.columns[12:])
+    # df = df.drop(range(1500000, 12591077))
+    # df = df.dropna()
+    # df = df.drop_duplicates()
+    # df = df.head(1000)
```

Comment:
Do we need these lines or can we delete?

Reply:
they show how to reproduce the sample dataset we provide from the kaggle dataset. I added a comment with that explanation